LayoutTests/fast/webgpu/nocrash/fuzz-283595-for-expected.txt

```diff
@@ -0,0 +1 @@
+Test passes if it does not crash.
```

Comment:
I don't think the expectations are correct?
The test cases should end with context loss if the browser works..

Reply:
Why should the context get lost? Because context is lost when the device is destroyed?

Reply:
I will move these under `/regression/` path instead since `/nocrash/` does not check the expectations

Reply:
I don't see a GPU restart while running the test locally, though I imagine that could occur on some devices.

I think the driver forward progress / infinite loop / hang detection mechanisms are not kicking in, so it thinks it's just a really long shader as the command buffer's completion handler never gets called when I run the test locally. I see the command buffer get scheduled, but it never completes after over a minute, this was on an M2 Mac Studio. The system is otherwise responsive, so it seems the preemption mechanisms are working even with this shader stuck in an infinite loop.

Reply:
I used a fragment shader instead which still triggers the GPU hang error and await the device lost promise in the expectations. Vertex shaders in WebGPU don't support writable storage buffers and while the repro could be modified to not use storage buffers, it wasn't immediately obvious how to test the out of bounds write did not occur.